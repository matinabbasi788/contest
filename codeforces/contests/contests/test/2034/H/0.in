3
5
2 4 6 8 10
5
12 15 21 30 35
3
2 3 6
