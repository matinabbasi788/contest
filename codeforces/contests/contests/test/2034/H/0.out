2
4 6
3
35 21 30
2
2 3